src/hooks/useConnectCurrentVenueNG.ts

```diff
@@ -17,7 +17,7 @@ export const currentVenueEventsNGSelector: SparkleSelector<
   Record<string, VenueEvent> | undefined
 > = (state) => state.firestore.data.currentVenueEventsNG;
 
-export const useConnectCurrentVenueNG = (venueId?: string) => {
+export const useConnectCurrentVenueNG = (venueId: string) => {
   useFirestoreConnect(
     !!venueId
       ? [
```

Comment:
Since `venueId` has been changed to always be required, this 'is defined' logic is no longer needed.

Reply:
as per https://github.com/sparkletown/sparkle/pull/1047#discussion_r561599060, I can't really control the absence of venueId outside of this hook, so I will probably leave it with `venueId` optional. The reason I have changed that is that I tried to use it in some place and just called it as is. I didn't get an error by calling it with an empty args list. So, I think I will wrap venueId into `options` object.

Reply:
You wouldn't have gotten an error when calling it with an empty args list because the only arg is allowed to be `undefined`. Which is sort of by definition how it works. Wrapping a single param function with an `options` object seems like overkill. The solution here would just be to ensure that you know what the code you are using takes as parameters/when/why, reading any documentation comments they have, etc. I don't believe we need to over-engineer a protection around this case, just need to be attentive to the code we are using, and how it should be used.